sherpa/__init__.py

```diff
@@ -116,11 +116,25 @@ def test(level=1, verbosity=1, datadir=None):
 
 
 def clitest():
-    try:
-        import pytest, pytest_cov
-        errno = pytest.main(['--pyargs', 'sherpa', '-rs'])
-        sys.exit(errno)
-    except ImportError:
-        print """Cannot import pytest and pytest-cov.
-            Please run 'pip install -r test_requirements.txt' first"""
-        sys.exit(1)
+    def install(package_name):
+        try:
```

Comment:
all exceptions are caught and then re-raised after the message has been printed, so the actual exception is displayed to the user.